app/src/components/common/form/date_field/index.tsx

```diff
@@ -137,7 +137,7 @@ export const DateField = (props: Props) => {
         calendarClassName="customCalendar"
         dateFormat="MMMM d, yyyy h:mm aa"
         disabled={disabled}
-        minDate={minDate}
+        minDate={new Date()}
```

Comment:
This is giving a warning because you're destructuring `minDate` from props and is not used afterward, maybe to maintain this prop as optional you can give a default value to it (`new Date()`) and maintain the possibility of passing a `minDate` different from now.

Reply:
added back!